packages/entities/entities-shared/src/components/entity-link/EntityLink.vue

```diff
@@ -15,8 +15,9 @@
     <KExternalLink
       v-else-if="entityUuid"
       class="entity-link"
+      :hide-icon="newWindow ? false : true"
       :href="externalLink"
-      target="_blank"
+      :target="newWindow ? '_blank' : '_self'"
```

Comment:
changed the window=blank prop to `newWindow`, since `external` was too close to `externalLink`